clang/test/Sema/warn-lifetime-analysis-nocfg.cpp

```diff
@@ -793,3 +806,202 @@ void test13() {
 }
 
 } // namespace GH100526
+
+namespace lifetime_capture_by {
+struct S {
+  const int *x;
+  void captureInt(const int&x [[clang::lifetime_capture_by(this)]]) { this->x = &x; }
```

Comment:
Be consistent regarding spacing around &.

Reply:
Done.